sql/migrations/20210105165912_world.sql

```diff
@@ -0,0 +1,502 @@
+(1083, 1, -11635.7, 639.049, 51.2367, 100, 0, 0, 0),
+(1083, 2, -11661, 651.638, 50.314, 100, 0, 0, 0),
+(1083, 3, -11666.9, 656.82, 50.2728, 100, 0, 0, 0),
+(1083, 4, -11669.7, 659.428, 50.3576, 100, 0, 0, 0),
+(1083, 5, -11673.7, 660.269, 50.3672, 100, 0, 0, 0),
+(1083, 6, -11689.8, 659.41, 48.8159, 100, 0, 0, 0),
+(1083, 7, -11694.7, 664.103, 49.5748, 100, 0, 0, 0),
+(1083, 8, -11697.1, 675.72, 50.0499, 100, 0, 0, 0),
+(1083, 9, -11698, 683.971, 49.9584, 100, 0, 0, 0),
+(1083, 10, -11701, 691.059, 49.7927, 100, 0, 0, 0),
+(1083, 11, -11708.1, 693.173, 49.8422, 100, 0, 0, 0),
+(1083, 12, -11719.6, 687.203, 49.7369, 100, 0, 0, 0),
+(1083, 13, -11716.4, 689.693, 49.8087, 100, 0, 0, 0),
+(1083, 14, -11709.4, 693.075, 49.8599, 100, 0, 0, 0),
+(1083, 15, -11701.9, 691.876, 49.7889, 100, 0, 0, 0),
+(1083, 16, -11698.2, 685.021, 50.0309, 100, 0, 0, 0),
+(1083, 17, -11697.1, 672.994, 50.355, 100, 0, 0, 0),
+(1083, 18, -11693.6, 661.737, 49.0561, 100, 0, 0, 0),
+(1083, 19, -11683.1, 659.888, 49.2773, 100, 0, 0, 0),
+(1083, 20, -11675.1, 660.256, 50.4412, 100, 0, 0, 0),
+(1083, 21, -11671.1, 659.864, 50.4003, 100, 0, 0, 0),
+(1083, 22, -11667.7, 658.053, 50.2973, 100, 0, 0, 0),
+(1083, 23, -11661.1, 651.681, 50.308, 100, 0, 0, 0);
+
+-- add creature 2174 waypoints
+UPDATE `creature` SET `position_x` = -11650.2, `position_y` = 616.899, `position_z` = 50.3042,  `movement_type` = 2 WHERE `guid` = 2174;
+INSERT INTO `creature_movement` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `waittime`, `wander_distance`, `script_id`) VALUES
+(2174, 1, -11650.2, 616.899, 50.3042, 100, 0, 0, 0),
+(2174, 2, -11657.2, 621.042, 50.5979, 100, 0, 0, 0),
+(2174, 3, -11668.3, 626.285, 50.2964, 100, 0, 0, 0),
+(2174, 4, -11680, 636.904, 49.7819, 100, 0, 0, 0),
+(2174, 5, -11690.9, 639.303, 49.311, 100, 0, 0, 0),
+(2174, 6, -11691.8, 629.401, 50.2928, 100, 0, 0, 0),
+(2174, 7, -11698.1, 616.096, 49.9239, 100, 0, 0, 0),
+(2174, 8, -11730.1, 599.91, 50.3767, 100, 0, 0, 0),
+(2174, 9, -11745.8, 596.134, 50.6429, 100, 0, 0, 0),
+(2174, 10, -11753.9, 595.691, 51.3053, 100, 0, 0, 0),
+(2174, 11, -11762.3, 599.841, 58.0965, 100, 0, 0, 0),
+(2174, 12, -11768, 603.493, 62.3273, 100, 0, 0, 0),
+(2174, 13, -11781.6, 603.359, 69.0334, 100, 0, 0, 0),
+(2174, 14, -11774.3, 604.712, 65.8086, 100, 0, 0, 0),
+(2174, 15, -11771, 604.588, 63.7455, 100, 0, 0, 0),
+(2174, 16, -11759.7, 597.777, 54.9956, 100, 0, 0, 0),
+(2174, 17, -11753.5, 595.697, 51.1528, 100, 0, 0, 0),
+(2174, 18, -11737.4, 597.525, 50.3733, 100, 0, 0, 0),
+(2174, 19, -11703.6, 611.221, 49.9859, 100, 0, 0, 0),
+(2174, 20, -11695.4, 619.649, 50.1001, 100, 0, 0, 0),
+(2174, 21, -11692.8, 637.087, 49.4544, 100, 0, 0, 0),
+(2174, 22, -11686.5, 639.636, 49.4809, 100, 0, 0, 0),
+(2174, 23, -11674.2, 631.017, 50.3015, 100, 0, 0, 0),
+(2174, 24, -11664.2, 624.674, 50.5842, 100, 0, 0, 0);
+
+-- remove incorrect creatures
+DELETE FROM `creature` WHERE `guid` IN (994, 993);

```

Comment:
You first give waypoints to this creature (203 line), and later you remove it. (GUID: 993)

Reply:
Thanks